src/odemis/model/_metadata.py

```diff
@@ -70,8 +70,7 @@
 MD_PIXEL_DUR = "Pixel duration"  # Time duration of a 'pixel' along the time dimension
 MD_TIME_OFFSET = "Time offset"  # Time of the first 'pixel' in the time dimension (added to ACQ_DATE), default is 0
 
-# TODO: MD_ACQ_TYPE: the type of acquisition contained in the DataArray, such as
-# EM_SPACIAL, FLUO_SPACIAL, ANCHOR_REGION, SPECTRUM, ANGULAR_RESOLVED...
+MD_ACQ_TYPE = "Acquisition type"  # the type of acquisition contained in the DataArray, such as EM_SPACIAL, FLUO_SPACIAL, ANCHOR_REGION, SPECTRUM, ANGULAR_RESOLVED...
```

Comment:
You need to also provide the (first) acquisition types as constants (and remove the example names in the comment). Letting them open-coded will too easily lead to bugs due to miss-spelling. See MD_DT_* for example.
I'd recommend:
MD_AT_SPECTRUM = "Spectrum"
MD_AT_AR = "Angle-resolved"
MD_AT_EM = "Electron microscope"
MD_AT_FLUO = "Fluoresence"
MD_AT_ANCHOR = "Anchor region"
MD_AT_OVERVIEW = "Overview"